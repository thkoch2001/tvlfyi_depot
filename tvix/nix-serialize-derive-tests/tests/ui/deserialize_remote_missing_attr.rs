use nix_compat_derive::nix_deserialize_remote;

pub struct Value(String);
impl From<String> for Value {
    fn from(s: String) -> Value {
        Value(s)
    }
}

nix_deserialize_remote!(
    #[nix()]
    Value
);

fn main() {}
