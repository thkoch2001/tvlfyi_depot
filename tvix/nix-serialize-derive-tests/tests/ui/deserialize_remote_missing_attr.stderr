error: Missing from_str, from or try_from attribute
  --> tests/ui/deserialize_remote_missing_attr.rs:10:25
   |
10 | nix_deserialize_remote!(#[nix()] Value);
   |                         ^^^^^^^^^^^^^^
